airflow/www/extensions/init_appbuilder.py

```diff
@@ -172,10 +180,13 @@ def init_app(self, app, session):
         app.config.setdefault("APP_ICON", "")
         app.config.setdefault("LANGUAGES", {"en": {"flag": "gb", "name": "English"}})
         app.config.setdefault("ADDON_MANAGERS", [])
+        app.config.setdefault("RATELIMIT_ENABLED", True)
```

Comment:
Not sure why this one is here, but it is one of the things the AppBuilder in FAB has comparing to us cc: @bolkedebruin  - maybe you know how RATELIMIT_ENABLED relates to AUTH_RATE_LIMITED?

Reply:
cc: @dpgaspar ?

Reply:
If I'm not mistaken, `RATELIMIT_ENABLED` will broadly enable rate limit since it can be added on `ModelRestApi` classes and `AUTH_RATE_LIMITED` will need `RATELIMIT_ENABLED` and add it to the auth views